mlflow/genai/judges/prompts/correctness.py

```diff
@@ -30,6 +32,15 @@
 
 If the claim is fully supported by the document in the context of the question, you must say "The response is correct" in the rationale. If the claim is not fully supported by the document in the context of the question, you must say "The response is not correct"."""  # noqa: E501
 
+CORRECTNESS_TRACE_FALLBACK = f"""{CORRECTNESS_BASE_INSTRUCTIONS}
```

Comment:
```suggestion
CORRECTNESS_TRACE_FALLBACK_INSTRUCTIONS = f"""{CORRECTNESS_BASE_INSTRUCTIONS}
```

Nit - appending _INSTRUCTIONS to the name is consistent with the approach for the non-fallback prompt, and it makes it clearer that this isn't the full fallback prompt (because `CORRECTNESS_PROMPT_OUTPUT` is appended later)